sopel/modules/reddit.py

```diff
@@ -236,9 +236,58 @@ def comment_info(bot, trigger, match):
     bot.say(message)
 
 
+def subreddit_sorting(bot, trigger, s, sorting):
+    if sorting == 'new':
+        submissions = list(s.new())
```

Comment:
The default `limit` in praw's `ListingGenerator` is 100. That's a bit much.

```suggestion
        submissions = list(s.new(limit=10))
```